### ISSUE TITLE
my model is useless,mAP=0

### ISSUE THREAD

**Issue by tttttthy** [fizyr/keras-retinanet#821]
Hello,I meet a question：
I trained my model on pascal voc2007 and my own datasets,my loss is 4 ,but my mAP is 0.The model is useless.I don't know why.（my keras vision is 2.2.4,my tensorflow-gpu vision is 1.1.0）.Please help me ,thanks a lot！
9963/9963 [==============================] - 3168s 318ms/step - loss: 4.0085 - regression_loss: 2.8106 - classification_loss: 1.1980
Running network: 100% (4952 of 4952) |###| Elapsed Time: 0:08:50 Time:  0:08:50
Parsing annotations: 100% (4952 of 4952) || Elapsed Time: 0:00:01 Time:  0:00:01
311 instances of class aeroplane with average precision: 0.0000
389 instances of class bicycle with average precision: 0.0000
576 instances of class bird with average precision: 0.0000
393 instances of class boat with average precision: 0.0000
657 instances of class bottle with average precision: 0.0000
254 instances of class bus with average precision: 0.0000
1541 instances of class car with average precision: 0.0000
370 instances of class cat with average precision: 0.0000
1374 instances of class chair with average precision: 0.0000
329 instances of class cow with average precision: 0.0000
299 instances of class diningtable with average precision: 0.0000
530 instances of class dog with average precision: 0.0000
395 instances of class horse with average precision: 0.0000
369 instances of class motorbike with average precision: 0.0000
5227 instances of class person with average precision: 0.0000
592 instances of class pottedplant with average precision: 0.0000
311 instances of class sheep with average precision: 0.0000
396 instances of class sofa with average precision: 0.0000
302 instances of class train with average precision: 0.0000
361 instances of class tvmonitor with average precision: 0.0000
mAP: 0.0000

**Comment by hgaiser** [comment]
Try inspecting your dataset using `debug.py`, it sounds like your annotations might be incorrect.

Closing this as it isn't an issue with the codebase. Please use the #keras-retinanet slack channel for support.
